src/Entity/Keyword.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Entity;
+
+use App\Enum\KeywordType;
+use App\Repository\KeywordRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Keywords class for Standardized keywords.
+ */
+#[ORM\Entity(repositoryClass: KeywordRepository::class)]
+class Keyword extends Entity
+{
+    /**
+     * JSON containing Standard Keyword Data.
+     */
+    #[ORM\Column]
+    private array $json = [];
```

Comment:
Does should allow for `mixed` unless you turn every json into an `array` which seems not practical.